petabvis/window_functionality.py

```diff
@@ -133,19 +156,37 @@ def table_tree_view(window: QtWidgets.QMainWindow, folder_path):
                 df = petab.get_observable_df(folder_path + "/" + filename)
                 if is_first_df:
                     window.observable_df = df
-            file.setData(df, role=Qt.UserRole + 1)
+            file.setData({"df": df, "name": filename}, role=Qt.UserRole+1)
```

Comment:
There's a few instances of `Qt.UserRole+1` in the code, and the `+1` is not so clear. Maybe a method like the following could make it more readable but I'm not so familiar with this code.
```python
def get_role():
    """
    Explanation of the +1.
    """
    return Qt.UserRole+1
```

Reply:
As I understand it, the +1 is just an arbitrary positive number so it does not have any special meaning (+2 and so on should work in just the same way)

Reply:
It's meant for users to define their own roles. The precise value does not matter, but it has to be >= Qt.UserRole. Ideally, create some constant with a descriptive name for that somewhere.